ZaeXT/test-scripts/test_failures.py

```
@@ -0,0 +1,183 @@
+
+    # 3. Alice 创建一个对话
+    res_alice_conv = requests.post(f"{BASE_URL}/conversations", headers=alice_headers, json={})
+    if res_alice_conv.status_code != 200 or "data" not in res_alice_conv.json():
+        print_fail("Alice 创建对话失败，无法继续进行权限测试", res_alice_conv)
+    alice_conv_id = res_alice_conv.json()["data"]["id"]
+    print(f"  -> Alice 创建了对话 (ID: {alice_conv_id})")
+
+    # --- 后续的核心测试逻辑保持不变 ---
+    # 4. 【核心测试】Bob 尝试删除 Alice 的对话
+    print(f"  -> 尝试用 Bob 的身份删除 Alice 的对话 (ID: {alice_conv_id})")
+    res_bob_delete = requests.delete(f"{BASE_URL}/conversations/{alice_conv_id}", headers=bob_headers)
```

Ruff (0.14.1)

Probable use of `requests` call without timeout

(S113)